js/plugins/drupalentity/plugin.js

```diff
@@ -166,7 +163,7 @@
       editor.on('doubleclick', function (evt) {
         var element = getSelectedEmbeddedEntity(editor) || evt.data.element;
 
-        if (isEmbeddedEntityWidget(editor, element)) {
+        if (isEditableEntityWidget(editor, element)) {
           editor.execCommand('editdrupalentity');
```

Comment:
You could also call `evt.cancel()` here so, you're sure that nothing else will handle this event.